resource_booking_combination_auto_create/wizards/resource_booking_combination_wizard.py

```diff
@@ -0,0 +1,209 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import _, api, fields, models
+
+
+class ResourceBookingCombinationWizard(models.TransientModel):
+    _name = "resource.booking.combination.wizard"
+    _description = "Resource Booking Combination Wizard"
+    _inherit = ["multi.step.wizard.mixin"]
+
+    resource_booking_id = fields.Many2one(
+        "resource.booking",
+        "Resource Booking",
+        required=True,
+        ondelete="cascade",
+    )
+    resource_booking_category_selection_ids = fields.One2many(
+        "resource.booking.category.selection",
+        "resource_booking_combination_wizard_id",
+        "Resource Categories",
+    )
+    current_resource_booking_category_selection_id = fields.Many2one(
+        "resource.booking.category.selection",
+        "Current Resource Category Selection",
+        ondelete="cascade",
+    )
+    current_resource_booking_category_selection_id_id = fields.Integer(
+        "Current Resource Category Selection ID",
+        related="current_resource_booking_category_selection_id.id",
+    )
+    current_resource_category_name = fields.Char(
+        related="current_resource_booking_category_selection_id"
+        ".resource_category_id.name",
+    )
+    available_resource_ids = fields.Many2many(
+        "resource.booking.category.selection.resource",
+        string="Resources",
+        related="current_resource_booking_category_selection_id.resource_ids",
+        readonly=False,
+    )
+    configure_step = fields.Integer("Configuration Step", default=0)
+    configure_step_count = fields.Integer(
+        "Number of Configuration Steps",
+        compute="_compute_configure_step_count",
+        store=True,
+    )
+    configure_step_message = fields.Char(
+        "Current Configure Step message",
+        compute="_compute_configure_step_message",
+    )
+    selected_resource_ids = fields.Many2many(
+        "resource.resource",
+        string="Selected Resources",
+        compute="_compute_selected_resource_ids",
+    )
+
+    @api.depends("resource_booking_category_selection_ids")
+    def _compute_configure_step_count(self):
+        self.configure_step_count = len(self.resource_booking_category_selection_ids)
```

Comment:
i think you’re right. however, i don’t know whether it could happen that a recordset of a wizard model refers to multiple records. i prefer to leave the code as is, as it’s more readable. if the problem ever happens, we’ll fix it.